babel/src/test/java/org/apache/calcite/test/BabelParserTest.java

```diff
@@ -328,6 +328,54 @@ private void checkParseInfixCast(String sqlType) {
     sql(sql).ok(expected);
   }
 
+  @Test public void testTableAttributeJournalDefault() {
+    final String sql = "create table foo, journal (bar integer)";
+    final String expected = "CREATE TABLE `FOO`, JOURNAL (`BAR` INTEGER)";
+    sql(sql).ok(expected);
+  }
+
+  @Test public void testTableAttributeJournalBefore() {
```

Comment:
testTableAttributeJournalBefore can be change to testTableAttributeJournal_defaultModifier_before